applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -757,11 +807,55 @@ int le_audio_volume_mute(void)
 
 int le_audio_play(void)
 {
+	int ret;
+
+	playing_state = true;
+
+	if (audio_streams[AUDIO_CH_L].ep->status.state == BT_AUDIO_EP_STATE_QOS_CONFIGURED) {
+		ret = bt_audio_stream_enable(&audio_streams[AUDIO_CH_L],
+					     lc3_preset_nrf5340.codec.meta,
+					     lc3_preset_nrf5340.codec.meta_count);
+
+		if (ret) {
+			LOG_WRN("Failed to enable left stream");
+		}
+	}
+
+	if (audio_streams[AUDIO_CH_R].ep->status.state == BT_AUDIO_EP_STATE_QOS_CONFIGURED) {
+		ret = bt_audio_stream_enable(&audio_streams[AUDIO_CH_R],
+					     lc3_preset_nrf5340.codec.meta,
+					     lc3_preset_nrf5340.codec.meta_count);
+
+		if (ret) {
+			LOG_WRN("Failed to enable right stream");
+		}
+	}
+
```

Comment:
Should we return 0 even if there might me errors?

Reply:
The problem is that you can't return the error from the first one without also trying to enable the second one. It's the same issue with sending to multiple streams. I figured as long as we print the WRN we at least see it in the log